app/controllers/application_controller.rb

```diff
@@ -41,7 +41,13 @@ def auth_member!
   end
 
   def auth_activated!
-    redirect_to settings_path, alert: t('private.settings.index.auth-activated') unless current_user.activated?
+    unless current_user.activated?
+      if current_user.email && !current_user.email_activated
+        redirect_to settings_path, alert: t('private.settings.index.auth-activated')
+      else current_user.phone_number && !current_user.phone_number_activated
```

Comment:
`else` should be `elsif`?

Can we add some controller spec to cover this logic?